Makefile

```diff
@@ -42,11 +42,11 @@ LIBRARIES += httpstan/lib/libprotobuf-lite.so
 endif
 INCLUDES_STAN_MATH_LIBS := httpstan/include/lib/boost_$(BOOST_VERSION) httpstan/include/lib/eigen_$(EIGEN_VERSION) httpstan/include/lib/sundials_$(SUNDIALS_VERSION) httpstan/include/lib/tbb_$(TBB_VERSION)
 INCLUDES_STAN := httpstan/include/stan httpstan/include/stan/math $(INCLUDES_STAN_MATH_LIBS)
-INCLUDES := httpstan/include/google/protobuf httpstan/include/pybind11 $(INCLUDES_STAN)
+INCLUDES := httpstan/include/google httpstan/include/pybind11 $(INCLUDES_STAN)
```

Comment:
Do we have more than 1 lib in google folder?

Reply:
`httpstan/include/google` is consistent with `httpstan/include/pybind`, the source code is included with <google/protobuf/...>.

Another problem here is that "INCLUDES" is not a good variable name since it does not give us anything equivalent to `include_dirs` in distutils terms.